2014-05-27 12:28:07.281 DEBUG nova.openstack.common.lockutils [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] Got semaphore "f1aed45d-76d4-40de-9a28-16cabbae8f2a" lock /opt/stack/nova/nova/openstack/common/lockutils.py:168
2014-05-27 12:28:07.282 DEBUG nova.openstack.common.lockutils [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] Got semaphore / lock "do_terminate_instance" inner /opt/stack/nova/nova/openstack/common/lockutils.py:248
2014-05-27 12:28:07.284 DEBUG nova.openstack.common.lockutils [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] Got semaphore "<function _lock_name at 0xa09110c>" lock /opt/stack/nova/nova/openstack/common/lockutils.py:168
2014-05-27 12:28:07.287 DEBUG nova.openstack.common.lockutils [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] Got semaphore / lock "_clear_events" inner /opt/stack/nova/nova/openstack/common/lockutils.py:248
2014-05-27 12:28:07.288 DEBUG nova.openstack.common.lockutils [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] Semaphore / lock released "_clear_events" inner /opt/stack/nova/nova/openstack/common/lockutils.py:252
2014-05-27 12:28:07.437 AUDIT nova.compute.manager [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] [instance: f1aed45d-76d4-40de-9a28-16cabbae8f2a] Terminating instance
2014-05-27 12:28:07.643 DEBUG nova.objects.instance [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] Lazy-loading `system_metadata' on Instance uuid f1aed45d-76d4-40de-9a28-16cabbae8f2a obj_load_attr /opt/stack/nova/nova/objects/instance.py:519
2014-05-27 12:28:08.214 DEBUG nova.network.api [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] Updating cache with info: [VIF({'ovs_interfaceid': None, 'network': Network({'bridge': u'br100', 'subnets': [Subnet({'ips': [FixedIP({'meta': {}, 'version': 4, 'type': u'fixed', 'floating_ips': [], 'address': u'10.0.0.6'})], 'version': 4, 'meta': {u'dhcp_server': u'10.0.0.1'}, 'dns': [IP({'meta': {}, 'version': 4, 'type': u'dns', 'address': u'8.8.4.4'})], 'routes': [], 'cidr': u'10.0.0.0/24', 'gateway': IP({'meta': {}, 'version': 4, 'type': u'gateway', 'address': u'10.0.0.1'})}), Subnet({'ips': [], 'version': None, 'meta': {u'dhcp_server': None}, 'dns': [], 'routes': [], 'cidr': None, 'gateway': IP({'meta': {}, 'version': None, 'type': u'gateway', 'address': None})})], 'meta': {u'tenant_id': None, u'should_create_bridge': True, u'bridge_interface': u'eth0'}, 'id': u'eca36745-dc0b-4981-a754-2f1ad1885d46', 'label': u'private'}), 'devname': None, 'qbh_params': None, 'meta': {}, 'details': {}, 'address': u'fa:16:3e:af:db:3d', 'active': False, 'type': u'bridge', 'id': u'fba595ef-5f3c-4dab-a527-c3b6fc9cbff1', 'qbg_params': None})] update_instance_cache_with_nw_info /opt/stack/nova/nova/network/api.py:74
2014-05-27 12:28:09.942 13476 DEBUG nova.virt.driver [-] Emitting event <nova.virt.event.LifecycleEvent object at 0xa4ae0cc> emit_event /opt/stack/nova/nova/virt/driver.py:1207
2014-05-27 12:28:09.942 13476 INFO nova.compute.manager [-] Lifecycle event 1 on VM f1aed45d-76d4-40de-9a28-16cabbae8f2a
2014-05-27 12:28:09.966 13476 INFO nova.virt.libvirt.driver [-] [instance: f1aed45d-76d4-40de-9a28-16cabbae8f2a] Instance destroyed successfully.
2014-05-27 12:28:10.082 DEBUG nova.virt.libvirt.vif [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] vif_type=bridge instance={'vm_state': u'active', 'availability_zone': u'nova', 'terminated_at': None, 'ephemeral_gb': 0, 'instance_type_id': 6, 'user_data': None, 'cleaned': False, 'vm_mode': None, 'deleted_at': None, 'reservation_id': u'r-nec4cjgc', 'id': 17, 'security_groups': [{'project_id': u'fe65afb7f564450dbdcb81ebabd50677', 'user_id': u'642f49ee797c4b329db7fd795f6c2645', 'description': u'default', 'deleted': False, 'created_at': datetime.datetime(2014, 4, 4, 10, 28, 24, tzinfo=<iso8601.iso8601.Utc object at 0x93747ac>), 'updated_at': None, 'deleted_at': None, 'id': 1, 'name': u'default'}], 'disable_terminate': False, 'display_name': u'mysecondinstance', 'uuid': 'f1aed45d-76d4-40de-9a28-16cabbae8f2a', 'default_swap_device': None, 'info_cache': {'instance_uuid': 'f1aed45d-76d4-40de-9a28-16cabbae8f2a', 'deleted': False, 'created_at': datetime.datetime(2014, 5, 9, 7, 1, 36, tzinfo=<iso8601.iso8601.Utc object at 0x93747ac>), 'updated_at': datetime.datetime(2014, 5, 9, 7, 1, 40, tzinfo=<iso8601.iso8601.Utc object at 0x93747ac>), 'network_info': [VIF({'ovs_interfaceid': None, 'network': Network({'bridge': u'br100', 'subnets': [Subnet({'ips': [FixedIP({'meta': {}, 'version': 4, 'type': u'fixed', 'floating_ips': [], 'address': u'10.0.0.6'})], 'version': 4, 'meta': {u'dhcp_server': u'10.0.0.1'}, 'dns': [IP({'meta': {}, 'version': 4, 'type': u'dns', 'address': u'8.8.4.4'})], 'routes': [], 'cidr': u'10.0.0.0/24', 'gateway': IP({'meta': {}, 'version': 4, 'type': u'gateway', 'address': u'10.0.0.1'})}), Subnet({'ips': [], 'version': None, 'meta': {u'dhcp_server': None}, 'dns': [], 'routes': [], 'cidr': None, 'gateway': IP({'meta': {}, 'version': None, 'type': u'gateway', 'address': None})})], 'meta': {u'tenant_id': None, u'should_create_bridge': True, u'bridge_interface': u'eth0'}, 'id': u'eca36745-dc0b-4981-a754-2f1ad1885d46', 'label': u'private'}), 'devname': None, 'qbh_params': None, 'meta': {}, 'details': {}, 'address': u'fa:16:3e:af:db:3d', 'active': False, 'type': u'bridge', 'id': u'fba595ef-5f3c-4dab-a527-c3b6fc9cbff1', 'qbg_params': None})], 'deleted_at': None}, 'hostname': u'mysecondinstance', 'launched_on': u'sandesh-virtual-machine', 'display_description': u'mysecondinstance', 'key_data': None, 'kernel_id': u'c9f75ac7-6bbd-4c17-801f-78525291d9b0', 'power_state': 1, 'default_ephemeral_device': None, 'progress': 0, 'project_id': u'fe65afb7f564450dbdcb81ebabd50677', 'launched_at': datetime.datetime(2014, 5, 9, 7, 1, 46, tzinfo=<iso8601.iso8601.Utc object at 0x93747ac>), 'config_drive': u'', 'node': u'sandesh-virtual-machine', 'ramdisk_id': u'0f964a81-814e-4229-9170-d056f452cc6f', 'access_ip_v6': None, 'access_ip_v4': None, 'deleted': False, 'key_name': None, 'updated_at': datetime.datetime(2014, 5, 27, 6, 58, 6, tzinfo=<iso8601.iso8601.Utc object at 0x93747ac>), 'host': u'sandesh-virtual-machine', 'ephemeral_key_uuid': None, 'architecture': None, 'user_id': u'642f49ee797c4b329db7fd795f6c2645', 'system_metadata': {u'image_kernel_id': u'c9f75ac7-6bbd-4c17-801f-78525291d9b0', u'instance_type_memory_mb': u'64', u'instance_type_swap': u'0', u'instance_type_vcpu_weight': None, u'instance_type_root_gb': u'0', u'instance_type_id': u'6', u'image_ramdisk_id': u'0f964a81-814e-4229-9170-d056f452cc6f', u'instance_type_name': u'm1.nano', u'instance_type_ephemeral_gb': u'0', u'instance_type_rxtx_factor': u'1.0', u'image_disk_format': u'ami', u'network_allocated': u'True', u'instance_type_flavorid': u'42', u'image_container_format': u'ami', u'instance_type_vcpus': u'1', u'image_min_ram': u'0', u'image_min_disk': u'0', u'image_base_image_ref': u'2ab6d5c8-9b0b-488a-993d-645d51bd76f2'}, 'task_state': u'deleting', 'shutdown_terminate': False, 'cell_name': None, 'root_gb': 0, 'locked': False, 'name': 'instance-00000011', 'created_at': datetime.datetime(2014, 5, 9, 7, 1, 36, tzinfo=<iso8601.iso8601.Utc object at 0x93747ac>), 'locked_by': None, 'launch_index': 0, 'metadata': {}, 'memory_mb': 64, 'vcpus': 1, 'image_ref': u'2ab6d5c8-9b0b-488a-993d-645d51bd76f2', 'root_device_name': u'/dev/vda', 'auto_disk_config': True, 'os_type': None, 'scheduled_at': datetime.datetime(2014, 5, 9, 7, 1, 36, tzinfo=<iso8601.iso8601.Utc object at 0x93747ac>)} vif=VIF({'ovs_interfaceid': None, 'network': Network({'bridge': u'br100', 'subnets': [Subnet({'ips': [FixedIP({'meta': {}, 'version': 4, 'type': u'fixed', 'floating_ips': [], 'address': u'10.0.0.6'})], 'version': 4, 'meta': {u'dhcp_server': u'10.0.0.1'}, 'dns': [IP({'meta': {}, 'version': 4, 'type': u'dns', 'address': u'8.8.4.4'})], 'routes': [], 'cidr': u'10.0.0.0/24', 'gateway': IP({'meta': {}, 'version': 4, 'type': u'gateway', 'address': u'10.0.0.1'})}), Subnet({'ips': [], 'version': None, 'meta': {u'dhcp_server': None}, 'dns': [], 'routes': [], 'cidr': None, 'gateway': IP({'meta': {}, 'version': None, 'type': u'gateway', 'address': None})})], 'meta': {u'tenant_id': None, u'should_create_bridge': True, u'bridge_interface': u'eth0'}, 'id': u'eca36745-dc0b-4981-a754-2f1ad1885d46', 'label': u'private'}), 'devname': None, 'qbh_params': None, 'meta': {}, 'details': {}, 'address': u'fa:16:3e:af:db:3d', 'active': False, 'type': u'bridge', 'id': u'fba595ef-5f3c-4dab-a527-c3b6fc9cbff1', 'qbg_params': None}) unplug /opt/stack/nova/nova/virt/libvirt/vif.py:774
2014-05-27 12:28:10.120 DEBUG nova.openstack.common.lockutils [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] Got semaphore "iptables" lock /opt/stack/nova/nova/openstack/common/lockutils.py:168
2014-05-27 12:28:10.121 DEBUG nova.openstack.common.lockutils [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] Attempting to grab file lock "iptables" lock /opt/stack/nova/nova/openstack/common/lockutils.py:178
2014-05-27 12:28:10.121 DEBUG nova.openstack.common.lockutils [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] Got file lock "iptables" at /opt/stack/data/nova/nova-iptables lock /opt/stack/nova/nova/openstack/common/lockutils.py:206
2014-05-27 12:28:10.121 DEBUG nova.openstack.common.lockutils [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] Got semaphore / lock "_apply" inner /opt/stack/nova/nova/openstack/common/lockutils.py:248
2014-05-27 12:28:10.122 DEBUG nova.openstack.common.processutils [req-4d54efe7-5f9d-4b07-98ff-730df9a9292f admin demo] Running cmd (subprocess): sudo nova-rootwrap /etc/nova/rootwrap.conf iptables-save -c execute /opt/stack/nova/nova/openstack/common/processutils.py:154
2014-05-27 12:28:10.421 13476 ERROR nova.virt.driver [-] Exception dispatching event <nova.virt.event.LifecycleEvent object at 0xa4ae0cc>: Info cache for instance f1aed45d-76d4-40de-9a28-16cabbae8f2a could not be found.
Traceback (most recent call last):

  File "/opt/stack/nova/nova/conductor/manager.py", line 597, in _object_dispatch
    return getattr(target, method)(context, *args, **kwargs)

  File "/opt/stack/nova/nova/objects/base.py", line 151, in wrapper
    return fn(self, ctxt, *args, **kwargs)

  File "/opt/stack/nova/nova/objects/instance.py", line 500, in refresh
    self.info_cache.refresh()

  File "/opt/stack/nova/nova/objects/base.py", line 151, in wrapper
    return fn(self, ctxt, *args, **kwargs)

  File "/opt/stack/nova/nova/objects/instance_info_cache.py", line 103, in refresh
    self.instance_uuid)

  File "/opt/stack/nova/nova/objects/base.py", line 112, in wrapper
    result = fn(cls, context, *args, **kwargs)

  File "/opt/stack/nova/nova/objects/instance_info_cache.py", line 70, in get_by_instance_uuid
    instance_uuid=instance_uuid)

InstanceInfoCacheNotFound: Info cache for instance f1aed45d-76d4-40de-9a28-16cabbae8f2a could not be found.
